src/components/Common.js

```diff
@@ -41,13 +42,19 @@ export const Button = styled.a`
 
   ${p =>
     p.theme.custom_imgs === 'hc' &&
-    css`
+    `
       background: url(${buttonBG});
       background-size: contain;
       background-repeat: no-repeat;
     `}
 `
 
+export const Button = props => (
```

Comment:
what is this change doing exactly?

Reply:
Buttons without `href` do not change pointer to cursor and do not appear clickable/have good keyboard accessibility. This change is to add a default `href`

Reply:
the more you know! thanks

Reply:
Actually this is an `a` element. Not exactly sure what happens if it were a `button` elem but what i said is defs true for `a`s